src/middleware/services.ts

```diff
@@ -18,6 +18,8 @@ export const initServices = (req: Request, res: Response, next: NextFunction): v
         res.locals.referrer = req.get('Referrer');
         res.locals.parseISO = parseISO;
         res.locals.dateFormat = format;
+        res.locals.dateAdd = add;
+        res.locals.languague = req.language;
```

Comment:
typo - but do we actually need this? If it's for the templates then it already exists on `i18n.language` in the views.

Reply:
Removed... Can't see where its used any more.